odoo/http.py

```diff
@@ -170,6 +170,7 @@
 mimetypes.add_type('application/font-woff', '.woff')
 mimetypes.add_type('application/vnd.ms-fontobject', '.eot')
 mimetypes.add_type('application/x-font-ttf', '.ttf')
+mimetypes.add_type('image/webp', '.webp')
```

Comment:
Do you confirm some OS don't know this mimetype and that we have to manually add it ?

Reply:
Both the OS used in runbot and the one on my local PC's (Linux Mint 20.2) do lack it.
Missing it makes `mimetypes.guess_type()` not return the mime type for the `.webp` extension, which is an issue in `_compute_mimetype()` of `ir.attachment`.